src/main/scala/ch/openolitor/core/db/evolution/scripts/v1/V1Scripts.scala

```diff
@@ -200,6 +200,8 @@ object V1Scripts {
         plz_lieferung varchar(10),
         ort_lieferung varchar(50),
         zusatzinfo_lieferung varchar(100),
+        lat_lieferung DECIMAL(9,6),
```

Comment:
Those fields where added in a change script later. Why do we need to add them to the initial script?
src/main/scala/ch/openolitor/core/db/evolution/scripts/v2/OO199_Coordinates.scala

Reply:
It was done like this before. Works only for optional fields.